lineage/figures/figure4.py

```diff
@@ -1,133 +1,108 @@
-""" This file contains functions for plotting different phenotypes in the manuscript. """
-
-import math
+""" This file is to show the model works in case we have rare phenotypes. """
 import numpy as np
 import pandas as pd
+import scipy.stats as sp
+
 import seaborn as sns
 from .figureCommon import (
     getSetup,
     subplotLabel,
+    commonAnalyze,
     pi,
     T,
```

Comment:
I'd recommend removing the import for `T`, just to make sure it's not conflicting with the T's in your for loop.